ledger/test-common/BUILD.bazel

```diff
@@ -109,6 +112,12 @@ da_scala_library(
         ),
         ("performance", []),
         ("package_management", []),
+        ("simple_package_party", []),
```

Comment:
@SamirTalwar-DA I know you wanted to move these to `kvutils` and I tested it but then I would have to publish `bazel_tools`

Reply:
What do you mean by "publish"?

Reply:
It would have to go to Artifactory

Reply:
As I would have to add the DARs as bazel `data` and access with `rlocation`.

Reply:
Why can't we package them inside JARs just as we do here?

Reply:
I can do it but then I'm wondering whether it's more reasonable to increase encapsulation or reduce duplication.

Reply:
By duplication I mean `BUILD.bazel` files and I won't be able to reuse the `TestDarReader` in this shape. Please let me know which approach is better.

Reply:
I think I'm gonna need @cocreature to weigh in here.

Reply:
I agree with @SamirTalwar-DA here. If we want to avoid duplication, we can factor it out in a macro that we share between here and `//kvutils/something`. I also prefer embedding it in JARs over publishing the dars directly.

Reply:
will do